src/documentation/pages/MessageBanner/examples/MessageBannerExampleJS.hbs

```diff
@@ -2,9 +2,20 @@
   var MessageBannerExample = document.querySelector('.docs-MessageBannerExample');
   var MessageBanner = new fabric['MessageBanner'](MessageBannerExample.querySelector('.ms-MessageBanner'));
   var MessageBannerButton = MessageBannerExample.querySelector('.docs-MessageBannerExample-button');
-  
+  var MessageBannerCloseButton = MessageBannerExample.querySelector('.ms-MessageBanner-close');
+
   // When clicking the button, open the MessageBanner
   MessageBannerButton.onclick = function() {
     MessageBanner.showBanner();
+    this.style.display = 'none';
```

Comment:
Let's add a class instead. We should never directly manipulate styles with JavaScript instead use a class like "is-hidden" or something.